lib/elixir/pages/Writing Documentation.md

```diff
@@ -51,6 +51,28 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
       size
     end
 
+## Documentation metadata
+
+Elixir allows developers to attach random metadata to the documentation too. This is done by using the relevant attribute, such as `@moduledoc`, `@typedoc` and `@doc` and passing a keyword list where the key is the metadata key and value is the metadata value. A commonly used metadata is `:since`, which annotates which version that particular module, function, type or callback was added. For instance, in the example above we could have added:
```

Comment:
```suggestion
Elixir allows developers to attach random metadata to the documentation too. This is done by using the relevant attribute (such as `@moduledoc`, `@typedoc`, and `@doc`) and passing a keyword list where the key is the metadata key and the value is the metadata value. A commonly used metadata is `:since`, which annotates in which version that particular module, function, type, or callback was added. For instance, in the example above we could have added:
```